_posts/2024-02-27-comet-donation.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Announcing Apache Arrow DataFusion Comet"
+date: "2024-02-27 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Introduction
+The Apache Arrow PMC is pleased to announce the donation of the [Comet project],
+a native Spark SQL Accelerator built on [Apache Arrow DataFusion].
+
+Comet is an Apache Spark Accelerator that uses Apache Arrow DataFusion to
```

Comment:
Maybe "Comet is an Apache Spark Accelerator" -> "Comet is an Apache Spark plugin"